lib/attack.go

```diff
@@ -11,6 +11,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/gavv/monotime"
```

Comment:
Is `monotime` used for compatibility reasons ? (because, as stated in their docs, since Go 1.9 standard time package uses monotonic clock when necessary - e.g. in `func Since(t Time) Duration`)

Reply:
Does the PR description answer your question?

Reply:
Not quite, looking at the code for the first time I can't see right away where `Attacker` is marshalled.

It seems to me that there is no marshalling of `Attacker` between `vegeta.NewAttacker(...)` and `a.hit(...)` calls, hence it should be fine to use `res.Timestamp = a.began.Add(time.Since(a.began))` instead of `res.Timestamp = a.began.Add(monotime.Since(a.monotonic))`.